ansible-scylla-monitoring/tasks/generate_scylla_servers.yml

```diff
@@ -5,10 +5,11 @@
   args:
```

Comment:
{{ scylla_api_address }}:{{ scylla_api_port }}

is also not part of monitoring role ...

Reply:
IF we are going to move to use https://github.com/scylladb/scylla-monitoring/blob/master/genconfig.py
then those params are likely not needed and inventory can be used, also minigenconfig.py will be not needed (it is buggy anyways)

Reply:
above said ... it's script Glauber wrote a LONG time ago, so it would be good to review it, it should work, but we should be sure and ideally @amnonh should bless its usage

Reply:
AH, they are hardcoded above ... OK ... likely not what we want

Reply:
I replaced the usage of `minigenconfig.py` with `genconfig.py`, as requested